src/metabase/search/postgres/index.clj

```diff
@@ -19,6 +21,14 @@
 
 (def ^:private tsv-language "english")
 
+(defn- random-prefix []
+  (str/replace (str (name *active-table*) "_" (random-uuid)) #"-" "_"))
+
+(defn random-table-name
```

Comment:
looks like it's only used in tests ,maybe move it there?

Reply:
I've put it here as there are plans to replace the 3-table switcharoo with random tables and an atom pointing at the active table.